README.md

```diff
@@ -153,6 +153,17 @@ order to use the enhanced commands, you must configure the keybindings yourself.
   )
 ~~~
 
+If you're using [Straight.el](https://github.com/raxod502/straight.el), install like so:
+
+~~~elisp
+(straight-use-package '(consult :type git :host github :repo "minad/consult"))
+(consult-annotate-mode)
+(setf (alist-get 'execute-extended-command consult-annotate-alist) #'consult-annotate-command-full)
+~~~
+
+You will need to add the key bindings yourself.
```

Comment:
Thank you! This is good addition for the readme.

Two things:
1. Can you please squash your commits such that it is only one commit in the end before merging? I prefer that for simple additions.
2. Can you rewrite the config a little bit, such a you mark somehow e.g. with an ellipsis where the use-package configuration from above should go? Such that we don't have to document things twice and such that the configurations do not differ? You added this setf as an example, I suppose? I've also seen there is something like ":straight t" for use-package. Could something like that be used? I am not a straight user myself, so I cannot tell what the best way is.

Reply:
I personally don't use `use-package` any more; just pure `straight.el`. There *is* a way to use `straight.el` with `use-package`; I'll add that example.

Reply:
@ashton314 Another question since I am using package.el+use-package.el - are there more package managers to consider or is straight the next most important package manager after package.el+elpa/melpa? I know that doom (and maybe spacemacs?) use straight.el, so I probably has a large user base. But I would like to avoid adding instructions for all the different possible setups. It is certainly okay to document the two or three most used alternatives. Selectrum is a bit more bold in that regards, but it is also by the author of straight.el, so this is understandable :)

Reply:
> I personally don't use use-package any more; just pure straight.el. There is a way to use straight.el with use-package; I'll add that example.

Ok, could explain this a bit better. I see use-package for configuration and package managers as orthogonal concerns. What I've read there seems to be mostly agreement regarding this. And since use-package is maintained by John Wiegley, will be upstreamed to Emacs proper soon, works great for me (one of the best packages behind the scenes for me to get some structure into my init.el). Therefore I would prefer if the configuration here also uses use-package. Is there something else which straight offers instead?

Reply:
That's a good question. My evolution through emacs packages has been built-in package.el -> use-package -> straight.el; I might not be the best person to ask what the most popular packages are.

> Melpa
Is this on Melpa now? Straight wasn't able to find the package on Melpa last time I tried. If that's the case, I'll revise the instructions to not worry about the git repo.

> Selectrum is a bit more bold in that regardsa

😃 That's for sure.

Reply:
> That's a good question. My evolution through emacs packages has been built-in package.el -> use-package -> straight.el; I might not be the best person to ask what the most popular packages are.

Ok, but I guess straight is good to have here! But since you don't use use-package anymore I would like to hear why!

> Is this on Melpa now? Straight wasn't able to find the package on Melpa last time I tried. If that's the case, I'll revise the instructions to not worry about the git repo.

No, the Melpa pipeline is somehow clogged unfortunately. I just asked on reddit https://www.reddit.com/r/emacs/comments/k5ebpj/is_there_any_chance_to_add_more_maintainers_to/

Reply:
@ashton314 It seems raxod502 also uses use-package, so we should use that too in any case here. See https://github.com/raxod502/radian/blob/06f2a30258deacb719a195221747ce1d528fc2fe/emacs/radian.el#L540